src/pages/Swap/AdvancedSwapMode/AdvancedSwapMode.styles.ts

```diff
@@ -0,0 +1,182 @@
+import styled, { css } from 'styled-components'
+
+export const BaseWrapper = styled.div`
+  background: #101019;
+  border-radius: 12px;
+  padding: 10px;
+`
+export const AdvancedModeDetailsItems = css`
+  & > div:nth-child(1),
+  & > div:nth-child(2) {
+    flex-basis: 25%;
+  }
+  & > div:nth-child(3) {
+    flex-basis: 20%;
+  }
+  & > div:nth-child(4) {
+    flex-basis: 10%;
+  }
+  & > div {
+    line-height: 1.4;
+  }
+`
+
+const TitleBase = css`
+  font-weight: 600;
+  font-size: 14px;
+  color: ${({ theme }) => theme.text4};
+  text-transform: uppercase;
+`
+
+export const AdvancedSwapModeContainer = styled.div`
+  backdrop-filter: blur(10px);
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 0.9fr)) minmax(0, 1.1fr);
+  gap: 15px;
+  padding: 0 20px;
+`
+export const PairDetailsWrapper = styled(BaseWrapper)`
+  grid-column: 1 / -1;
+  padding: 15px 20px;
+`
+export const PairSymbols = styled.div`
+  color: ${({ theme }) => theme.text2};
+  font-weight: 700;
+  font-size: 16px;
+  margin: 0 5px;
+`
+
+export const PairInfo = styled.div`
+  width: 10%;
+  margin-left: 20px;
+  overflow: hidden;
+
+  & > div:nth-child(2) {
+    margin-top: 10px;
+  }
+`
+
+export const PairTab = styled.div<{ size?: string }>`
+  color: ${({ theme }) => theme.purple3};
+  font-size: ${({ size }) => (size ? size : '11px')};
+  font-weight: 500;
+  letter-spacing: 0.08em;
+`
+
+export const PairValue = styled.div`
+  font-size: 14px;
+  font-weight: 600;
+  color: ${({ theme }) => theme.text3};
+`
+
+export const PairValueChange = styled(PairValue)<{ positive: boolean; size?: string; space?: boolean }>`
+  color: ${({ positive }) => (positive ? '#0E9F6E' : '#F02E51')};
```

Comment:
use theme colors, this actually theme.green2 and theme.red1

```suggestion
  color: ${({ positive, theme }) => (positive ? theme.green2 : theme.red1)};
```